llama-index-integrations/readers/llama-index-readers-file/llama_index/readers/file/rtf/base.py

```diff
@@ -12,7 +12,7 @@ class RTFReader(BaseReader):
     def load_data(
         self,
         input_file: Union[Path, str],
-        extra_info=Dict[str, Any],
+        extra_info=Dict[str, Any] | None = None,
```

Comment:
Oh we should use `Optional[Dict[str, Any]]` instead, need to be safe for python3.8